src/lib/installation/cio_ignore.rb

```diff
@@ -255,15 +240,22 @@ def write_autoconf_setting
     def add_cio_boot_kernel_parameters
       Yast.import "Bootloader"
 
+      param = Yast::Bootloader.kernel_param(:common, "cio_ignore")
+
+      return unless param == :missing
+
+      res = Yast::WFM.Execute(YAST_LOCAL_BASH_PATH, "/sbin/cio_ignore -k")
+
+      raise "cio_ignore -k failed with #{res["stderr"]}" if res["exit"] != 0
+
       # boot code is already proposed and will be written in next step, so just modify
-      Yast::Bootloader.modify_kernel_params("cio_ignore" => "all,!ipldev,!condev")
+      Yast::Bootloader.modify_kernel_params("cio_ignore" => res["stdout"].lines.first)
```

Comment:
please strip lines here. I am not sure how bootloader handle newline in string

```suggestion
      Yast::Bootloader.modify_kernel_params("cio_ignore" => res["stdout"].lines.first.strip)
```